xbmc/cores/RetroPlayer/cheevos/Cheevos.cpp

```diff
@@ -0,0 +1,159 @@
+/*
+ *  Copyright (C) 2020-2021 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "Cheevos.h"
+
+#include "URL.h"
+#include "filesystem/CurlFile.h"
+#include "filesystem/File.h"
+#include "games/addons/GameClient.h"
+#include "games/addons/cheevos/GameClientCheevos.h"
+#include "utils/JSONVariantParser.h"
+#include "utils/URIUtils.h"
+#include "utils/Variant.h"
+#include "utils/auto_buffer.h"
+#include "utils/log.h"
+
+using namespace KODI;
+using namespace RETRO;
+
+// API JSON Field names
+constexpr auto SUCCESS = "Success";
```

Comment:
Constants should go in an anonymous namespace, so that the symbols are private to this file and don't clash with other global symbols of the same name:

```c++
namespace
{
constexpr auto SUCCESS = "Success";
constexpr auto GAME_ID = "GameID";
constexpr auto PATCH_DATA = "PatchData";
constexpr auto RICH_PRESENCE = "RichPresencePatch";

constexpr int RESPORNSE_SIZE = 64;
}
```